### ISSUE TITLE
terraform-provider-cloudflare doesn't build with go 1.13

### ISSUE THREAD

**Issue by perosb** [terraform-providers/terraform-provider-cloudflare#482]
Hi,

I'm on archlinux which has go1.13.
It seem terraform-provider-cloudflare doesn't compile out of the box.

Tried 1.16 - 1.18.1.
1.16 has compiled fine before (probably on go1.12).

Error message:
`go: google.golang.org/genproto@v0.0.0-20190819205937-24fa4b261c55: invalid pseudo-version: does not match version-control timestamp (2019-08-19T20:19:41Z)`

If I manually change go.mod to the timestamp I get something similar from:
`go: github.com/cloudflare/cloudflare-go@v0.9.4 requires
        golang.org/x/lint@v0.0.0-20190511005446-959b441ac422: invalid pseudo-version: does not match version-control timestamp (2019-04-09T20:28:23Z)`

```
$ go version
go version go1.13 linux/amd64
```

```
$ go env
GO111MODULE=""
GOARCH="amd64"
GOBIN=""
GOCACHE="/home/perosb/.cache/go-build"
GOENV="/home/perosb/.config/go/env"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GONOPROXY=""
GONOSUMDB=""
GOOS="linux"
GOPATH="/home/perosb/go"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/usr/lib/go"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/usr/lib/go/pkg/tool/linux_amd64"
GCCGO="gccgo"
AR="ar"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD=""
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build527785514=/tmp/go-build -gno-record-gcc-switches"
```

**Comment by jacobbednarz** [comment]
This may have been fixed recently as I can't reproduce on MacOS 10.14.6.

```
$ echo "1.13.1" > .go-version

$ go version
go version go1.13.1 darwin/amd64

$ go get
go: downloading golang.org/x/net v0.0.0-20190930134127-c5a3c61f89f3
go: downloading github.com/cloudflare/cloudflare-go v0.10.2
go: downloading google.golang.org/genproto v0.0.0-20190927181202-20e1ac93f88c
go: extracting golang.org/x/net v0.0.0-20190930134127-c5a3c61f89f3
go: extracting github.com/cloudflare/cloudflare-go v0.10.2
go: downloading golang.org/x/time v0.0.0-20190921001708-c4c64cad1fd0
go: extracting golang.org/x/time v0.0.0-20190921001708-c4c64cad1fd0
go: extracting google.golang.org/genproto v0.0.0-20190927181202-20e1ac93f88c
go: finding github.com/cloudflare/cloudflare-go v0.10.2
go: finding golang.org/x/net v0.0.0-20190930134127-c5a3c61f89f3
go: finding google.golang.org/genproto v0.0.0-20190927181202-20e1ac93f88c
go: finding golang.org/x/time v0.0.0-20190921001708-c4c64cad1fd0

$ go build

$ file terraform-provider-cloudflare
terraform-provider-cloudflare: Mach-O 64-bit executable x86_64
```

Can you please try again with the latest master branch?

**Comment by perosb** [comment]
@jacobbednarz confirmed. It works on latest master.
